app/services/planning_application_creation_service.rb

```diff
@@ -43,6 +42,7 @@ def build_planning_application
 
     planning_application.assign_attributes(site_params) if site_params.present?
     planning_application.assign_attributes(result_params) if result_params.present?
+    planning_application.assign_attributes(boundary_geojson: params[:boundary_geojson]) if params[:boundary_geojson].present?
```

Comment:
Since we are updating to use json instead of the string I think we need to update this to use permitted params, outlining the nested structure we are expecting

Reply:
I've first updated it and deleted this line. But the `if` statement questioned me whether I had to add it on `permitted params` since we don't require it. Added it back now and kept the `if` statement